entrypoint.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+set -e
+ETC_HOSTS_PROXY_BIND_ADDRESS="${ETC_HOSTS_PROXY_BIND_ADDRESS:-0.0.0.0:8080}"
+ETC_HOSTS_PROXY_MODE="${ETC_HOSTS_PROXY_MODE:-http}"
+ADDITIONAL_ARGS=""
+
+if [ ! -z "ETC_HOSTS_PROXY_BIND_ADDRESS" ]; then
+  ADDITIONAL_ARGS="$ADDITIONAL_ARGS -L $ETC_HOSTS_PROXY_BIND_ADDRESS"
+fi
+
+if [ ! -z "ETC_HOSTS_PROXY_MODE" ]; then
+  ADDITIONAL_ARGS="$ADDITIONAL_ARGS -M $ETC_HOSTS_PROXY_MODE"
+fi
+
+if [ ! -z "$ETC_HOSTS_PROXY_HOSTS_LIST" ]; then
+  ADDITIONAL_ARGS="$ADDITIONAL_ARGS -H $ETC_HOSTS_PROXY_HOSTS_LIST"
+fi
+
+if [[ "$1" == "run" ]] && [[ $# -eq 1 ]]; then
+  if [[ -z "$ETC_HOSTS_PROXY_HOSTS_LIST" ]] ; then
+    echo "Add some hosts to the list or use full args in docker run: $0 run -M http -L 0.0.0.0:8080 -H <your_host>=<your_ip>"
+    exit -1
+  fi
+  echo "Start proxy with args: $0 run $ADDITIONAL_ARGS"
+  /usr/bin/etc-hosts-proxy "$@ $ADDITIONAL_ARGS"
```

Comment:
This doesn't work for me:
```
jay@lima-docker:~$ docker run -e ETC_HOSTS_PROXY_HOSTS_LIST="yandex.ru=127.0.0.1,google.com=127.0.0.1" -p 8080:8080 --rm -it etc-hosts-proxy:latest
Start proxy with args: /entrypoint.sh run  -L 0.0.0.0:8080 -M http -H yandex.ru=127.0.0.1,google.com=127.0.0.1
Error: unknown command "run  -L 0.0.0.0:8080 -M http -H yandex.ru=127.0.0.1,google.com=127.0.0.1" for "etc-hosts-proxy"
Run 'etc-hosts-proxy --help' for usage.
FATA[0000] unknown command "run  -L 0.0.0.0:8080 -M http -H yandex.ru=127.0.0.1,google.com=127.0.0.1" for "etc-hosts-proxy" 
```

`shellcheck` complains too:
> SC2145 (error): Argument mixes string and array. Use * or separate argument.

Let me remember how to deal with it properly.. there was some specific kind of string expansion related to arrays..

Reply:
yeah, sorry write on rush and without checks %) fixed